content/en/_TEMPLATE.md

```diff
@@ -4,9 +4,8 @@ status: Feedback Appreciated
 category: concept
 ---
 
-## What it is
 
-Quick summary of the concept and what it is.
```

Comment:
We can probably leave the "and what it is" line here as this is a section that is now the introduction.